modules/smart-agent_nagios-status-check/variables.tf

```diff
@@ -50,3 +50,14 @@ variable "status_check_lasting_duration_seconds" {
   default     = "900"
 }
 
+variable "runbook_url" {
```

Comment:
this is the object of another issue and so another PR which should be spread to the entire repo (not only one module) to keep modules homogeneous, same for `tip`

Reply:
Removed, i will try to create another PR for this